iphone/CoreApi/CoreApi/Bookmarks/MWMBookmarksManager.mm

```diff
@@ -754,5 +754,25 @@ - (void)resetElevationMyPositionChanged {
   self.bm.SetElevationMyPositionChangedCallback(nullptr);
 }
 
+- (dp::Color)getColorFromUIColor:(UIColor *)color {
+    CGFloat fRed = 0;
+    CGFloat fGreen = 0;
+    CGFloat fBlue = 0;
+    CGFloat fAlpha = 0;
```

Comment:
There is no need to initialize vars when they will be immediately re-initialized below. Please also try to follow 2 spaces indent.
```suggestion
- (dp::Color)getColorFromUIColor:(UIColor *)color {
  CGFloat fRed, fGreen, fBlue, fAlpha;
```